src/exporter.js

```diff
@@ -21,13 +20,10 @@ function Exporter (hash, dagService, options) {
     return new Exporter(hash, dagService, options)
   }
 
-  // Sanitize hash.
+  // Sanitize hash
   if (!isIPFS.multihash(hash)) {
     throw new Error('not valid multihash')
   }
-  if (Buffer.isBuffer(hash)) {
-    hash = bs58.encode(hash)
```

Comment:
Why was this removed? It breaks functionality. :(  Fixed in https://github.com/ipfs/js-ipfs-unixfs-engine/pull/52